src/components/molecules/UiFormField/UiFormField.vue

```diff
@@ -141,6 +141,8 @@ const defaultProps = computed(() => {
 const inputId = computed(() => (
   props.id || `input-${uid()}`
 ));
+const hintValue = computed(() => (typeof props.hint === 'string' ? props.hint : false));
```

Comment:
IMO we can try to add computed `hasHint`, `hasErrorMessage`
```
  const hasHint = computed(() => (typeof props.hint !== 'boolean'));
```
remember to bind hasHint to slot :)